### ISSUE TITLE
Can you provide the method to train using our own corpora using your version of fairseq ?

### ISSUE THREAD

**Issue by vishnu3741** [jerinphilip/ilmulti#8]
I normally use indicnlp to tokenize and moses to train the MT but your model is giving better accuracy and can you give an insight into the amount or corpus used to train the model? Thank you.

**Comment by jerinphilip** [comment]
Perhaps the paper linked below will answer the corpus used. 

* [Revisiting Low Resource Status of Indian Languages in Machine Translation](https://arxiv.org/abs/2008.04860).

Regarding the data/training:
* [fairseq/data/cvit/corpora.py](https://github.com/jerinphilip/fairseq-ilmt/blob/master/fairseq/data/cvit/corpora.py) has dataset with a tag based inclusion, which I specify through a configuration file ([example](https://github.com/jerinphilip/fairseq-ilmt/blob/pib/pib-exps/pib-train-config.yaml)). An example training script in our cluster looks like [this](https://github.com/jerinphilip/fairseq-ilmt/blob/pib/pib-exps/pib-train.sh).

**Comment by vishnu3741** [comment]
hey, Is there way to add vocabulary (I mean words) to the model instead of retraining the entire model? can we edit the files in mm-all-iter1 to do this?

**Comment by jerinphilip** [comment]
[This](https://kheafield.com/papers/edinburgh/transfer.pdf) paper might have some useful information, I think. I'd just retrain with new vocabulary, turnaround is approximately 1 day or something on 4 GPUs to start getting reasonable numbers. This one used 1080Tis or 2080Tis.
